pyproject.toml

```diff
@@ -1,3 +1,22 @@
 [build-system]
 requires = ["setuptools>=42"]
-build-backend = "setuptools.build_meta"
\ No newline at end of file
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "clmbr-altk"
```

Comment:
Change back jsut to altk for now, then ultk in a new branch etc, as discussed on Zulip

Reply:
Will do